test/nft_upload_test.py

```diff
@@ -11,10 +11,7 @@
 >>> g.plot()
 >>> NFT_STORAGE_API_KEY = os.environ["NFT_STORAGE_API_KEY"]
 >>> g.generate()
->>> random_projection = random.choice(list(Projection))
->>> random_color = random.choice(VALID_COLORS)
->>> random_bgcolor = random.choice(VALID_COLORS)
->>> g.plot(projection=random_projection,color=random_color,bgcolor=random_bgcolor)
+>>> g.plot(projection=Projection.RANDOM,color="random",bgcolor="random")
```

Comment:
I prefer `g.plot(projection=Projection.RANDOM, color="random", bgcolor="random")`. This way we could keep our test's code style consistent.

Reply:
fixed in https://github.com/sepandhaghighi/samila/pull/124/commits/43bae2e88f413befba40f81b4dcc892f1e7a6912